addons/sale_loyalty/views/templates.xml

```diff
@@ -46,7 +46,7 @@
                             <t t-set="gift_card" t-value="gift_cards[gift_card_idx]"/>
                             <td class="o_purchased_gift_card">
                                 Gift #<t t-esc="gift_card_idx + 1"/>
-                                (<span t-esc="order._get_real_points_for_coupon(gift_card, post_confirm=True)"/> <span t-esc="gift_card.point_name"/>)
+                                (<span t-esc="gift_card._format_points(order._get_real_points_for_coupon(gift_card, post_confirm=True), name='')"/><span t-esc="gift_card.point_name"/>)
```

Comment:
```suggestion
                                (<span t-esc="gift_card._format_points(order._get_real_points_for_coupon(gift_card, post_confirm=True))"/>)
```
IIUC, you are keeping those spans only in case someone targeted this element in an inherited view?
Because I'd say that we could drop them and only keep a simplified version of  `_format_points` :thinking:.